tests/pass-dep/pass-crates/shims/fs_with_isolation.stderr

```diff
@@ -0,0 +1,20 @@
+warning: `open` was made to return an error due to isolation
```

Comment:
Why is this a new file, not a moved file? Looks like there are a bunch of stale stderr/stdout files in this PR?

Reply:
oops, fixed